pkg/apis/pipeline/v1beta1/pipelinerun_types.go

```diff
@@ -511,9 +511,11 @@ type PipelineTaskRun struct {
 // PipelineTaskRunSpec  can be used to configure specific
 // specs for a concrete Task
 type PipelineTaskRunSpec struct {
-	PipelineTaskName       string       `json:"pipelineTaskName,omitempty"`
-	TaskServiceAccountName string       `json:"taskServiceAccountName,omitempty"`
-	TaskPodTemplate        *PodTemplate `json:"taskPodTemplate,omitempty"`
+	PipelineTaskName       string                   `json:"pipelineTaskName,omitempty"`
+	TaskServiceAccountName string                   `json:"taskServiceAccountName,omitempty"`
+	TaskPodTemplate        *PodTemplate             `json:"taskPodTemplate,omitempty"`
+	StepOverrides          []TaskRunStepOverride    `json:"stepOverrides,omitempty"`
+	SidecarOverrides       []TaskRunSidecarOverride `json:"sidecarOverrides,omitempty"`
```

Comment:
NIT: I assume that if a `PipelineRun` that specifies any of these was fetched via the `v1alpha1` API, it would lose them, which I think it probably fine, but I wonder if we should document somewhere clearly what are the fields in `v1beta` which are not in `v1alpha` and that would be lost when fetching/updating via `v1alpha1`. /cc @sbwsg @vdemeester 

Not something we need to fix as part of this PR, it just came to mind seeing the new fields 🙏